src/web/src/assets/dark.scss

```diff
@@ -53,6 +53,28 @@
     }
   }
 
+  .nav-link {

```

Comment:
Could you make a var(...) for these so they fit the styling of the rest of the file and can be more easily reused?